charts/invenio/values.yaml

```diff
@@ -105,6 +105,21 @@ invenio:
     secretKeys:
       usernameKey: "DATACITE_USERNAME"
       passwordKey: "DATACITE_PASSWORD"
+    ## @param invenio.datacite.prefix DataCite DOI prefix
+    ##
+    prefix: ""
+    ## @param invenio.datacite.testMode DataCite test mode enabled.
+    ## This has to be a string value, invenio will evaluate this as a python expresion an transform it into a boolean.
+    ## Example:
+    ## testMode: "False"
+    ##
+    testMode: ""
+    ## @param invenio.datacite.format A string used for formatting the DOI.
+    ##
+    format: ""
```

Comment:
I am not sure if we want to set defaults here, i.e. https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py#L466, or just check in the chart if the value is set (this is what I did)

Reply:
I believe it's a bit more than that. It actually formats the DOI. 
If you set it to `{prefix}/repo.{id}` the DOIs will look like `10.5281/repo.abc1-xyz9`.

The syntax is described on the link above, more or less 😅 You can see an example in action here https://github.com/zenodo/zenodo-rdm/blob/master/invenio.cfg#L327